package.json

```diff
@@ -22,8 +22,9 @@
     "vinyl-source-stream": "^1.1.0"
   },
   "scripts": {
+    "postinstall": "gem install html-proofer scss_lint",
     "start": "gulp && gulp clean-all && gulp website",
-    "build": "gulp production build:website",
+    "build": "gulp production build:website && htmlproofer ./public --disable-external",
```

Comment:
The build script now runs some `html-proofer` tests so we can have some good tests for link-regressions. The external link checker is disabled. You can run `npm run build` to test locally.